.github/workflows/client-compatibility-tests.yml

```
@@ -218,15 +216,15 @@ jobs:
           slots_per_epoch=2
           addresses_to_fund=["0xf39Fd6e51aad88F6F4ce6aB8827279cffFb92266"]
           EOF
-          
+
           BASE64_CONFIG_OVERRIDE=$(cat config.toml | base64 -w 0)
           echo ::add-mask::$BASE64_CONFIG_OVERRIDE
           echo "BASE64_CONFIG_OVERRIDE=$BASE64_CONFIG_OVERRIDE" >> $GITHUB_ENV
           touch .root_dir
       - name: Run Tests
         uses: smartcontractkit/chainlink-github-actions/chainlink-testing-framework/run-tests-binary@912bed7e07a1df4d06ea53a031e9773bb65dc7bd # v2.3.0
         with:
-          test_command_to_run: ./tests -test.timeout ${{ matrix.timeout }} -test.run ${{ matrix.test }}
+          test_command_to_run: ./tests -test.timeout ${{ matrix.timeout }} -test.parallel=2 -test.run ${{ matrix.test }}
```

```suggestion
          test_command_to_run: ./tests -test.timeout ${{ matrix.timeout }} -test.run ${{ matrix.test }}
```
We don't need this anymore if we intend on only running 1 test.